catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -224,4 +221,54 @@ private String transformToXml(String xhtml) {
             return xhtml;
         }
     }
+
+    /*
+        XML handler that outputs empty body.
```

Comment:
I would add a comment somewhere that the body contains the complete text of the original document and that this handler will only return metadata.